src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/PlacementRequestsTest.kt

```diff
@@ -1036,6 +1113,9 @@ class PlacementRequestsTest : IntegrationTestBase() {
             ).produce(),
         )
         withApArea(apArea)
+        apArea?.defaultCruManagementArea?.let {
+          withCruManagementArea(it)
+        }
```

Comment:
can we please add `cruManagementArea` as a first class parameter in the `createPlacementRequest() `function and use that here instead? You can create one in the calling tests using `Given an CAS1 CRU Management Area`()

Reply:
Added `cruManagementArea` to `createPlacementRequest()`.